src/multiselect-combo-box.html

```diff
@@ -7,42 +7,100 @@
 
 <dom-module id="multiselect-combo-box">
   <template>
-    <label part="label">[[label]]</label>
 
-    <div part="readonly-container" hidden$="[[!readonly]]">
-      [[_getReadonlyValue(selectedItems, itemLabelPath, compactMode)]]
-    </div>
+    <style>
+
+      :host {
+        display: inline-flex;
+        width: 100%;
+      }
+      
```

Comment:
Also here there is some trailing whitespaces, please remove them and all errors should be resolved. Thank you :)